appengine/logs/README.md

```diff
@@ -0,0 +1,39 @@
+# Users Authentication sample for Google App Engine
+
+This sample demonstrates how to use the [Logs API][appid] on [Google App
+Engine][ae-docs].
+
+[appid]: https://cloud.google.com/appengine/docs/java/logs/
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Running locally 
+
+The Logs API only generates output for deployed apps, so this program should not be run locally.
+
+## Deploying
+ 
+This example uses the
+[Maven gcloud plugin](https://cloud.google.com/appengine/docs/java/managed-vms/maven).
+
+In the following command, replace YOUR-PROJECT-ID with your
+[Google Cloud Project ID](https://developers.google.com/console/help/new/#projectnumber) and SOME-VERSION with the desired version number.
```

Comment:
Link seems broken to me when I try it. :-( Looks like the content moved to https://support.google.com/cloud/answer/6158840